src/Raven.Studio/typescript/commands/database/settings/deleteConnectionStringCommand.ts

```diff
@@ -4,12 +4,20 @@ import endpoints = require("endpoints");
 
 class deleteConnectionStringCommand extends commandBase {
 
-    constructor(private db: database, private type: Raven.Client.Documents.Operations.ConnectionStrings.ConnectionStringType, private connectionStringName: string) {
+    connectionStringType: Raven.Client.Documents.Operations.ConnectionStrings.ConnectionStringType;
+    
+    constructor(private db: database, private type: StudioEtlType, private connectionStringName: string) {
```

Comment:
can we use OngoingTask type here?

Reply:
Can't,
`OngoingTaskType` has 'QueueEtl'
The connection string type is `enum ConnectionStringType` with only 'Queue'

Reply:
refactored, as discussed